### PR TITLE
Added ormsgpack as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/help-python-c @conda-forge/help-c-cpp @conda-forge/help-python @conda-forge/help-rust not sure why win build is successful but not unix ones... Something about GCC overlinking?

**Comment by xhochy** [comment]
For macOS, you need to increase the deployment target to 10.12, but for linux, I'm also confused.

**Comment by thewchan** [comment]
@xhochy that's what I thought with OSX and I can do that.... But I don't get it for Linux

**Comment by dbast** [comment]
the error on linux is:
```ERROR (ormsgpack,lib/python3.12/site-packages/ormsgpack/ormsgpack.cpython-312-x86_64-linux-gnu.so): .. but ['conda-forge/linux-64::libgcc-ng==13.2.0=h807b86a_5'] not in reqs/run````

Does maybe adding `- {{ compiler('rust') }}` solve that as it adds libgcc-ng to the run-time requirements of the resulting package?

File: recipes/ormsgpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ormsgpack-{{ version }}.tar.gz
+  sha256: b0feb4b80b871df04bb5db79ff2ea1bee8a5483e70676387449631d778ad9b18
+
+build:
+  noarch: python
```
Comment (xhochy) [inline recipes/ormsgpack/meta.yaml:13]: This should be removed. This package is not noarch

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/ormsgpack/meta.yaml (hunk 1 above)
Comment (thewchan) [reply recipes/ormsgpack/meta.yaml]: Silly mistake on my part; didn't check this from grayskull. Thanks!

**Review by xhochy — APPROVED** [review]

**Pull request merged**